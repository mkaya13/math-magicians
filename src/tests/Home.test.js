import TestRenderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../components/Navbar';

describe('Test Home page', () => {
  it('should renders Home page', () => {
    const HomeComponent = TestRenderer.create(<Router><Home /></Router>).toJSON();
    expect(HomeComponent).toMatchSnapshot();
  });
});
