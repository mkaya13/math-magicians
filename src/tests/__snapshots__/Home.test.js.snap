// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home page should renders Home page 1`] = `
<nav
  className="navBar"
>
  <span>
    <h2
      className="logo"
    >
      Math Magicians
    </h2>
  </span>
  <ul>
    <li>
      <a
        aria-current="page"
        className="active-link active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="active-link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className="active-link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
